src/main.rs

```diff
@@ -79,6 +81,19 @@ use umode::UmodeTask;
 #[panic_handler]
 fn panic(info: &core::panic::PanicInfo) -> ! {
     println!("panic : {:?}", info);
+    if let Some(location) = info.location() {
+        println!(
+            "panic occurred in file '{}' at line {}",
+            location.file(),
+            location.line()
+        );
+    }
+
+    println!("panic backtrace:");
+    backtrace().skip(2).for_each(|frame| {
```

Comment:
put a comment about the `skip(2)`

Reply:
fixed

Reply:
My bad. I forgot to push out the latest version.